streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -78,6 +80,64 @@
         /*probingRebalanceIntervalMs*/ 60 * 1000L
     );
 
+    @Test
+    public void shouldBeStickyForActiveAndStandbyTasksEvenIfNoWarmups() {
```

Comment:
First test for stickiness: we should be 100% sticky and also not schedule a probing rebalance when we are configured for no warmups.